ci/build-linux.sh

```diff
@@ -22,15 +22,35 @@ if [ "${TRAVIS_OS_NAME}" != "linux" ]; then
 fi
 
 readonly IMAGE="cached-${DISTRO}-${DISTRO_VERSION}"
-echo "IMAGE = ${IMAGE}"
-
 sudo docker build -t "${IMAGE}:tip" \
      --build-arg DISTRO_VERSION="${DISTRO_VERSION}" \
      --build-arg CXX="${CXX}" \
      --build-arg CC="${CC}" \
      --build-arg NCPU="${NCPU:-2}" \
      --build-arg BUILD_TYPE="${BUILD_TYPE:-Release}" \
      --build-arg CHECK_STYLE="${CHECK_STYLE:-}" \
+     --build-arg SCAN_BUILD="${SCAN_BUILD:-}" \
      --build-arg GENERATE_DOCS="${GENERATE_DOCS:-}" \
      --build-arg CMAKE_FLAGS="${CMAKE_FLAGS:-}" \
      -f "ci/Dockerfile.${DISTRO}" .
+
+if [ "${SCAN_BUILD:-}" = "yes" ]; then
```

Comment:
2-space indent here as well, please — it's now codified in our `.editorconfig` so it's official! ;-)

Reply:
Yeah, and my IDE seems to forget about it every other day... grrrr.